caesars-palace/blocks/slider-card/slider-card.js

```diff
@@ -1,42 +1,75 @@
 export default function decorate(block) {
   const cardWrapper = document.createElement('div');
   cardWrapper.classList.add('card-wrapper');
-  block.querySelectorAll('div.slider-card > div').forEach((div) => {
+
+  function isADesktop() {
+    const mediaDesktop = window.matchMedia('only screen and (min-width: 769px)');
+    return mediaDesktop.matches;
+  }
+
+  // default value is 3
+  let numberOfCardsDisplayed = 3;
+  block.querySelectorAll('div.slider-card > div').forEach((div, idx) => {
+    // We do not need to add a class to the first div as it includes the number of
+    // cards we wish to display
+    if (idx === 0) {
+      numberOfCardsDisplayed = div.children[0].innerHTML;
```

Comment:
I'm not sure if this is the best way for an author to control a block. Would it be possible instead for them to add a variation to the block title that controls how many slides are visible?

eg.
```
slider-card  (max-visible-3)
```

Reply:
It's a personal preference, but I prefer authoring and defining as first line the number of visible slides. It makes it more obvious. Make ask them what they prefer?

Reply:
I've just never seen this done before on previous blocks. An author would need to know in this particular block usage that they need to add an extra row that will not be included as part of the content but needs to have an arbitrary number in it. Are there any other ways you have found on how blocks can be configured?

Reply:
Oh, we actually have something similar in the carousel to set the length a carousel slide is displayed. The idea was well received, so we did it that way. It helps them to have a good mental image of the site that's being built. I think doing this here with the first line being a simple configuration is a similar idea.

Reply:
If this is a pattern we want to promote then we could possibly add a new function in scripts that is similar to `readBlockConfiguration` that understands this pattern. Are more than one config values allowed? Could name value paris be introduced so that it is easier to author/read? Just having one number in a row makes to hard to understand its meaning.